### PR TITLE
supporting another H1-POS variant

### PR DESCRIPTION
**Pull request by makrsmark**

### CODE REVIEW

File: lib/plugins/Label_H1_POS.ts [hunk 1]
```diff
@@ -88,7 +87,42 @@ export class Label_H1_POS extends DecoderPlugin {
...
+        processPosition(decodeResult, field.substring(2));
+      } else {
+        if(i===0) {
+          processUnknown(decodeResult, field);
+        } else {
+          processUnknown(decodeResult, '/'+field);
+        }
+      }
+    }
+    processRoute(decodeResult, parts[7], parts[5], parts[9], parts[8], undefined, date);
```
Comment (makrsmark) [inline lib/plugins/Label_H1_POS.ts:114]: the timestamps might not be correct here. they definitely look like timestamps but are very far apart

File: lib/plugins/Label_H1_POS.test.ts [hunk 2]
```diff
@@ -324,6 +324,60 @@ test('decodes Label H1 Preamble #M1BPOS long variant', () => {
...
+
+  expect(decodeResult.decoded).toBe(true);
+  expect(decodeResult.decoder.decodeLevel).toBe('partial');
+  expect(decodeResult.decoder.name).toBe('label-h1-pos');
+  expect(decodeResult.formatted.description).toBe('Position Report');
+  expect(decodeResult.formatted.items.length).toBe(5);
+  expect(decodeResult.formatted.items[0].label).toBe('Aircraft Position');
+  expect(decodeResult.formatted.items[0].value).toBe('39.277 N, 77.359 W');
+  expect(decodeResult.formatted.items[1].label).toBe('Aircraft Route');
+  expect(decodeResult.formatted.items[1].value).toBe('(39.3 N, 77.11 W)@2024-03-03T14:28:00Z > (38.56 N, 77.15 W)@2024-03-03T03:14:30Z > ?');
```
Comment (makrsmark) [inline lib/plugins/Label_H1_POS.test.ts:344]: positions will need trailing zeros when #47 is merged

File: lib/plugins/Label_H1_POS.test.ts [hunk 3]
```diff
@@ -324,6 +324,60 @@ test('decodes Label H1 Preamble #M1BPOS long variant', () => {
...
+  const decodeResult = decoderPlugin.decode({ text: text });
+  console.log(JSON.stringify(decodeResult, null, 2));
+
+  expect(decodeResult.decoded).toBe(true);
+  expect(decodeResult.decoder.decodeLevel).toBe('partial');
+  expect(decodeResult.decoder.name).toBe('label-h1-pos');
+  expect(decodeResult.formatted.description).toBe('Position Report');
+  expect(decodeResult.formatted.items.length).toBe(5);
+  expect(decodeResult.formatted.items[0].label).toBe('Aircraft Position');
+  expect(decodeResult.formatted.items[0].value).toBe('42.579 N, 108.09 W');
```
Comment (makrsmark) [inline lib/plugins/Label_H1_POS.test.ts:369]: position will need trailing zero when https://github.com/airframesio/acars-decoder-typescript/pull/47 is merged

**Review by kevinelliott — APPROVED** [review]
Awesome!

**Pull request merged**